test/src/test/java/org/corfudb/infrastructure/CompactorServiceUnitTest.java

```diff
@@ -124,4 +125,38 @@ public void runOrchestratorLeaderTest() {
         verify(leaderServices).initCompactionCycle();
         verify(invokeCheckpointingJvm, times(1)).shutdown();
     }
+
+    @Test
+    public void runOrchestratorSchedulerTest() {
```

Comment:
It looks like these tests have a race between the time the mocks are prepared and the time in which the compactorService::start() method schedules the runOrchestrator thread. If the runOrchestrator thread makes it far enough into its execution, then this mocked behaviour will not have taken effect.

Reply:
Will look into this and fix it in a later CL